pkg/api/validation/validation.go

```diff
@@ -2348,9 +2348,16 @@ func ValidatePodUpdate(newPod, oldPod *api.Pod) field.ErrorList {
 		activeDeadlineSeconds := *oldPod.Spec.ActiveDeadlineSeconds
 		mungedPod.Spec.ActiveDeadlineSeconds = &activeDeadlineSeconds
 	}
+
+	// Allow tolerations updates.
+	if len(newPod.Spec.Tolerations) > 0 {
+		allErrs = append(allErrs, validateTolerations(newPod.Spec.Tolerations, fldPath.Child("tolerations"))...)
+	}
+	mungedPod.Spec.Tolerations = oldPod.Spec.Tolerations
```

Comment:
Just to be clear that as per @davidopp's comment, we are still going to allow modifications to `tolerationSeconds` to old/existing tolerations and also allowing any new additions.